spec/services/candidate_interface/submit_application_choice_spec.rb

```diff
@@ -74,6 +74,39 @@
           submit_application
         }.to have_enqueued_mail(ProviderMailer, :application_submitted)
       end
+
+      it 'duplicates the work experiences on the application_choice' do
+        work_experience = create(
+          :application_work_experience,
+          experienceable: application_form,
+        )
+
+        submit_application
+        expect(application_choice.work_experiences.count).to eq(1)
```

Comment:
Give that an `application_choice.work_experiences` will return the `application_form.application_work_experiences` if there are none could we add a check to make sure there are new records?
Comparing `id` would be useful and/or checking that `ApplicationWorkExperience.all.count` has changed 🤔

Reply:
`application_work_experiences` will return `application_form.application_work_experiences`, `work_experiences` will return `application_choice.work_experiences`

Will add the id comparison though and checking if it changed, that's a good shout